README.md

```diff
@@ -109,16 +109,22 @@ import (
 	"context"
 	"fmt"
 	"time"
-	
+
 	"github.com/tarantool/go-tarantool/v2"
 )
 
 func main() {
-	opts := tarantool.Opts{User: "guest"}
-	ctx, cancel := context.WithTimeout(context.Background(), 
+	dialer := tarantool.NetDialer {
+		Address: "127.0.0.1:3301",
+		User: 	 "guest",
+	}
+	opts := tarantool.Opts{
+		Timeout: time.Second,
+	}
+	ctx, cancel := context.WithTimeout(context.Background(),
 		500 * time.Millisecond)
 	defer cancel()
```

Comment:
Let's change the initialization order here and make it the same as for `tarantool.Connect()`: ctx, dialer, opts.

Reply:
Done.